### PR TITLE
doc: Jewel 10.2.6 release notes

### PR DESCRIPTION
**Pull request by theanalyst**
Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

### CODE REVIEW

**Comment by theanalyst** [comment]
@liewegas Do you think we have any pending major announcements worth mentioning in release notes

File: doc/release-notes.rst [hunk 1]
```diff
@@ -2514,6 +2514,209 @@ Notable Changes
 * tools: ceph-disk: change ownership of initfile to ceph:ceph (`issue#16280 <http://tracker.ceph.com/issues/16280>`_, `pr#9688 <http://github.com/ceph/ceph/pull/9688>`_, Shylesh Kumar)
 * test: ceph_test_rados_api_tmap_migrate: remove test for tmap_upgrade (`pr#10234 <http://github.com/ceph/ceph/pull/10234>`_, Kefu Chai)
 
+
+v10.2.6 Jewel
+=============
+
+This point release fixes several important bugs in RBD mirroring, RGW multi-site, CephFS, and RADOS.
+
+We recommend that all v10.2.x users upgrade. Also note the following when upgrading from hammer
```
Comment (liewegas) [inline doc/release-notes.rst]: Is this meant to be half a sentence?

I don't think there is anything major to include here...

Comment (theanalyst) [inline doc/release-notes.rst]: oops, fixing

Comment (theanalyst) [inline doc/release-notes.rst]: done

**Pull request merged**